Makefile

```diff
@@ -157,5 +157,6 @@ prefix=/usr
 # other requirements (correct ownership of files, etc.) is managed by
 # the service script on sudo service pihole-FTL (re)start
 install: pihole-FTL
-	install -m 0755 pihole-FTL $(prefix)/bin
-	/sbin/setcap CAP_NET_BIND_SERVICE,CAP_NET_RAW,CAP_NET_ADMIN+eip $(prefix)/bin/pihole-FTL
+	mkdir -p $(DESTDIR)$(prefix)/bin
```

Comment:
We previously assumed `/usr/bin` to always exist. With a custom `DESTDIR`, we can not be sure any longer if this is the case and, hence, we make parent directories as needed.